### DISCUSSION TITLE
Flutter integration tests don't see NyState screens

### DISCUSSION THREAD

**Discussion by Skylaunch** [nylo-core/nylo#173] [General]
Hello! 

I am currently trying to connect e2e tests to the flutter project using the documentation https://docs.flutter.dev/testing/integration-tests

However, I ran into the problem. 
The pages using NyState and rendered through "view" method do not render in tests, respectively we do not see any widgets.

I made a simple comparison: I wrote a screen with minimal functionality and wrapped it in the view method using NyState of NyStatefulWidget. The result: A blank screen in the tests and "find" object does not see anything. Then I replaced the NyState on State of StatefulWidget and everything worked correctly.

Thanks in advance for the assistance!

**Comment by agordn52** [comment]
Hello @Skylaunch,

Sorry for the late reply.
I'll look into this, I think I know what's happening.

**Reply by Skylaunch** [reply]
Hi! I'm still waiting for an answer...

Are you continuing to move in this direction?

**Reply by agordn52** [reply]
Hi @Skylaunch,

Sorry, I'll need a bit of time to look into this. 
If you have any ideas for a PR though, I'd be interested to see your approach.

**Comment by Skylaunch** [comment]
Hi, my problem is still relevant, may I know if the solution has progressed?

I'm sorry, but I don't have any PR ideas :(

**Reply by agordn52** [reply]
I'll need to add a test for testing NyState widgets. 

Since they use different methods from 'State', that's the reason it's not working.
